app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::GreetingsController < ApplicationController
+  def random
+    @random_greeting = Greeting.order('RANDOM()').first
+    render json: { content: @random_greeting.message }
+  end
+end
```

Comment:
![image](https://github.com/mariegrace31/hello-rails-back-end/assets/93039359/3eb0c9cc-ca08-45eb-bc9e-2826f674c5b5)

- Great work so far but I see a problem In your backend and front end because i am not able to run your app properly :fearful:

- [ ] In the backend, the column name is content not message to it would be great if you fix that name :+1:
- [ ] In the frontend, i have seen the message is not being rendered even after fixing the backend issue :fearful: Kindly fix these issues to make your app work as expected :+1: